data/deaths/2022.json

```
@@ -4205,6 +4205,39 @@
           "trustful": true
         }
       ]
+    },
+    {
+      "cause": "suicide",
+      "count": 1,
+      "published": true,
+      "orphans": 1,
+      "unpublished_reason": "",
+      "year": "2022",
+      "month": "09",
+      "day": "04",
+      "keywords": "Alain Tai",
+      "house": "pompier",
+      "peers": [],
+      "text": "Un Caporal Chef du CSP de Châtellerault a mis fin a ses jours.",
+      "section": "CPS de Châtellerault - 86",
+      "location": "Châtellerault",
+      "county": "86",
+      "origin": "interieur",
+      "gps": {
+        "accurate": false,
+        "radius": 150,
+        "lat": 46.8177334,
+        "lng": 0.5459586
+      },

```

```suggestion
      "gps": {
        "accurate": true,
        "radius": 0,
        "lat": 46.802518878848595,
        "lng": 0.5474193250918571
      },

```

Sauf gros évènement médiatique je ne localise plus directement le lieu du décès mais l'emplacement de son corps de rattachement (ici le CIS de Châtellerault)